pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -83,6 +83,14 @@ tasks:
       connections.outputName: diffimPlots
       connections.fakesType: parameters.fakesType
       connections.coaddName: parameters.coaddName
+      atools.dipoleQuiver: DiffimSpatialMetricsDipoleQuiverPlot
+      atools.dipoleQuiver.angleName: dipole_direction
+      atools.dipoleQuiver.lengthName: dipole_separation
+      atools.dipoleQuiver.produce.plot.qKeyLabel: "0.2 arcsec Avg Dipole length"
+      atools.kernelQuiver: DiffimSpatialMetricsDipoleQuiverPlot
+      atools.kernelQuiver.angleName: psfMatchingKernel_position_angle
```

Comment:
After thinking about this longer, I agree with you that this would make more sense in detector coordinates using `psfMatchingKernel_direction`, even if that is different from the dipole plots.
Ideally we would also have a marker indicating the direction towards zenith, but that seems beyond the scope of the current ticket.

Reply:
This is corrected now. Thanks.